### PR TITLE
Update to Robolectric 3.3.2.

### PR DESCRIPTION
**Pull request by NightlyNexus**

### CODE REVIEW

File: picasso/src/test/java/com/squareup/picasso/OkHttp3DownloaderTest.java [hunk 1]
```diff
@@ -15,21 +15,22 @@
  */
 package com.squareup.picasso;
 
-import okhttp3.*;
```
Comment (JakeWharton) [inline picasso/src/test/java/com/squareup/picasso/OkHttp3DownloaderTest.java]: Omg who let this guy in?

File: picasso/src/test/java/com/squareup/picasso/OkHttp3DownloaderTest.java [hunk 2]
```diff
@@ -15,21 +15,22 @@
...
+import org.robolectric.RobolectricTestRunner;
 import org.robolectric.annotation.Config;
 
 import static org.fest.assertions.api.Assertions.assertThat;
 
-@RunWith(RobolectricGradleTestRunner.class)
+@RunWith(RobolectricTestRunner.class)
 @Config(
     sdk = 23, // Works around https://github.com/robolectric/robolectric/issues/2566.
     shadows = { Shadows.ShadowNetwork.class }
```
Comment (vanniktech) [inline picasso/src/test/java/com/squareup/picasso/OkHttp3DownloaderTest.java]: This one should no longer be needed with Robolectric 3.3.2

**Comment by JakeWharton** [comment]
I vaguely remember not updating because they broke the shadow `Bitmap` behavior.

**Comment by vanniktech** [comment]
[Here's the commit](https://github.com/robolectric/robolectric/commit/2022e7b2ef8ba02101a4a09b823907022b40eb90)

**Comment by NightlyNexus** [comment]
Oops, yep, I'll look at this later today. (Battery just died.)

**Comment by NightlyNexus** [comment]
trash, i haven't been able to figure out how to fix it. sorry.

**Pull request closed without merging**